include/Util/TransparentImage.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef UTIL_TRANSPARENT_IMAGE_HPP
+#define UTIL_TRANSPARENT_IMAGE_HPP
+
+#include <string>
+#include <SDL.h>
+#include <SDL_image.h>
+#include <iostream>
+#include <Util/Logger.hpp>
+
+#include "Core/TransparentImageBase64.hpp"
+#include "Base64.hpp"
+
+static constexpr auto TRANSPARENT_IMAGE_BASE64_DECODE_LENGTH = DecodeBase64Length(TRANSPARENT_IMAGE);
+static constexpr auto TRANSPARENT_IMAGE_BASE64_DECODE = DecodeBase64<TRANSPARENT_IMAGE_BASE64_DECODE_LENGTH>(TRANSPARENT_IMAGE);
+
+SDL_Surface* GetTransparentImageSDLSurface(){
```

Comment:
This causes ODR warnings, you should move the definition into a `.cpp` file. Also, shouldn't this function be named `GetMissingTextureSdlSurface()` because missing does not mean transparent

Reply:
BTW `TRANSPARENT_IMAGE_BASE64_DECODE`, `TransparentImageBase64.hpp` should rename too